src/Language/PureScript/Docs/Convert.hs

```diff
@@ -97,7 +100,7 @@ augmentDeclarations (partitionEithers -> (augments, toplevels)) =
 addDefaultFixity :: Declaration -> Declaration
 addDefaultFixity decl@Declaration{..}
   | isOp declTitle && isNothing declFixity =
-        decl { declFixity = Just defaultFixity }
+        decl { declFixity = Just (defaultFixity, Nothing) }
```

Comment:
Just checking: once we require operators to be defined as aliases for other existing values, this `addDefaultFixity` function will no longer be necessary, right?